### ISSUE TITLE
1.17 support ETA?

### ISSUE THREAD

**Issue by SteveShady** [Pokechu22/WorldDownloader#224]
Roadmap or timeline for when to expect 1.17 support?

**Comment by uyjulian** [comment]
As usual, there is no ETA for next version support.

**Comment by Pokechu22** [comment]
It's probably going to take a while, though, as 1.17 has made a lot of internal changes (with things like support for variable world height, though they haven't actually changed the world height by default) that are likely to break existing assumptions.  Furthermore, my old laptop died a few weeks ago, and I don't have a lot of my development tools set up on this one (I also think that a lot of the development tools will need to be redone due to changes from 1.17, such as updating to Java 16).

Basically, it'll be ready when it's ready, and hopefully that won't be too long.

**Comment by dillfrescott** [comment]
Cant wait! I love this mod <3
